src/Goerli-DssSpell.sol

```diff
@@ -15,8 +15,8 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 pragma solidity 0.6.12;
-// Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
-// pragma experimental ABIEncoderV2;
+// Enable ABIEncoderV2 when onboarding collateral
```

Comment:
It's better to keep the original text because this doesn't apply when adding RWA collateral:

```suggestion
// Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
```